packages/graphback-cli/src/helpers/db.ts

```diff
@@ -52,14 +53,14 @@ export const createDBResources = async (configInstance: ConfigBuilder): Promise<
       await execa('touch', ['db.sqlite'])
     }
 
-    const schemaText: string = models.found.map((m: string) => readFileSync(`/${m}`, 'utf8')).join('\n')
-
-    const backend: GraphQLBackendCreator = new GraphQLBackendCreator(schemaText, graphqlCRUD)
+    const schemaContext = new LocalSchemaProvider(folders.migrations, folders.model)
+    const backend: GraphQLBackendCreator = new GraphQLBackendCreator(schemaContext, graphqlCRUD)
 
     const manager = new DatabaseSchemaManager(database, dbConfig);
+
     backend.registerDataResourcesManager(manager);
 
-    await backend.createDatabase()
+    await backend.migrateDatabase(initialization)
```

Comment:
Side note: Always add some info about breaking API (we have people using this outside cli so db and generate methods are part of the API)